week-4/D-methods/exercise-4.js

```diff
@@ -6,7 +6,10 @@ Define a method "makeFriend" to add a new friend to her list.
 
 var person = {
     name: "Alice",
-    friends: ["John", "Nina"]
+    friends: ["John", "Nina"],
+    makeFriend: function(newfriend) {
+        return this.friends.push(newfriend);
```

Comment:
Don't need `return` here.  We're not asked to have `makeFriend` function return anything